### DISCUSSION TITLE
Date faces applied to dates in link descriptions

### DISCUSSION THREAD

**Discussion by jkroes** [minad/org-modern#234] [Q&A] [answered]
I create org-roam notes with date titles quite frequently. When I link to those notes from other notes, the face `org-modern-date-inactive` is applied to them. 

<img width="731" alt="Screenshot 2024-09-14 at 12 34 20 PM" src="https://github.com/user-attachments/assets/7db1a1e8-4d81-44e4-b850-3577f38e770b">

**Comment by minad** [comment, marked as answer]
Hi @jkroes! Thanks for the report. This is a slightly unfortunate interaction. As a quick workaround, I suggest to disable `org-modern-timestamp` by setting it to nil. However this might not be a completely satisfactory solution. I'll try to add a check preventing date highlighting in links, and see if this works out.

EDIT: Potential fix in https://github.com/minad/org-modern/commit/bb048dcbcaae19762f0c1097d27027695512fb7b. Hope this works.

**Reply by jkroes** [reply]
Thank you! That fixed the issue on my end.
